scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala

```diff
@@ -47,23 +47,42 @@ class ProcessMetaDataBuilder private[build](metaData: MetaData) {
   }
 }
 
+object ScenarioBuilder {
+
+  def streaming(id: String) =
+    new ProcessMetaDataBuilder(MetaData(id, StreamMetaData()))
+
+  def streamingLite(id: String) =
+    new ProcessMetaDataBuilder(MetaData(id, LiteStreamMetaData()))
+
+  def requestResponse(id: String) =
+    new ProcessMetaDataBuilder(MetaData(id, RequestResponseMetaData(None)))
+
+}
+
+@deprecated("use ScenarioBuilder streaming method", "1.3")
 object EspProcessBuilder {
 
   def id(id: String) =
     new ProcessMetaDataBuilder(MetaData(id, StreamMetaData()))
 
 }
 
+@deprecated("use ScenarioBuilder streamingLite method", "1.3")
```

Comment:
IMO we can just remove those two builders - lite isn't used anywhere except our code

Reply:
yeah, you're right but to be consistent let's remove it in 1.4 version